### DISCUSSION TITLE
EvaluateScriptAsync but it's not working

### DISCUSSION THREAD

**Discussion by boinaweb** [cefsharp/CefSharp#4574] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

115.3.130

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.NetFramework 4.7.2

### Implementation

WinForms

### Reproduction Steps

EvaluateScriptAsync I use with instagram but it is not working when I login to the account, before logging in it injects normally but when logging in it stops working

### Expected behavior

In versions 108.4.130 it works normally

### Actual behavior

In versions 108.4.130 it works normally

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

### Other information

In versions 108.4.130 it works normally

**Comment by amaitland** [comment]
Converted to discussion as it's this is unlikey a bug in `CefSharp`.

> ### Reproduction Steps
> EvaluateScriptAsync I use with instagram but it is not working when I login to the account, before logging in it injects normally but when logging in it stops working

This really doesn't tell me anything meaningful. What does stops working actually mean? What error are you seeing? 

Check the log file:
https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file
Confirm your script actually runs via `DevTools`.
https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging

- Each major version comes with a different version of `Chromium`, different security policies.
- Major websites like `instagram` often block embedded browsers
